ui/js/controllers/logviewer.js

```diff
@@ -190,6 +190,15 @@ logViewerApp.controller('LogviewerCtrl', [
                     {label: "End", value: dateFilter(job.end_timestamp*1000, thDateFormat)}
                 ];
 
+                // Test to expose the reftest button in the logviewer actionbar
+                $scope.isReftest = function() {
+                    if (job.job_group_name.indexOf("Reftest") !== -1) {
```

Comment:
You might consider having this check include ``toLowerCase()`` like:
    job.job_group_name.toLowerCase().indexOf("Reftest")